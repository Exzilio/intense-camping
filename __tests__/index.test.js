import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import IndexPage from '../pages';

it('renders correctly', () => {
  const tree = renderer.create(
    <IndexPage/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});