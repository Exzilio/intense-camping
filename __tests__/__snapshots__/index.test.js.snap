// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="root"
  data-jsx={879608397}
>
  <h1
    data-jsx={879608397}
  >
    Intense Camping
  </h1>
  <h2
    data-jsx={879608397}
  >
    Book Your CHILL Trip
  </h2>
  <p
    data-jsx={879608397}
  >
    <a
      data-jsx={879608397}
      href="/trips"
      onClick={[Function]}
    >
      Sign-up Now
    </a>
  </p>
  <img
    alt="chill"
    data-jsx={879608397}
    src="/images/chill-camping.jpg"
  />
</div>
`;
